src/components/pages/Nav.css

```diff
@@ -0,0 +1,34 @@
+.subContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 10vh;
+  border-bottom: 2px solid rgb(80, 80, 80);
+  margin-bottom: 1rem;
+  padding: 0.3rem 5rem 0.2rem 3rem;
+}
+
+.logoContainer {
+  display: flex;
+  align-items: center;
+
+  /* background-color: lightgray; */
```

Comment:
Kindly remove this commented line of code to make your work cleaner.

Reply:
Hi @OyePriscilla, thanks for pointing that out. I will fix it now